lib/Doctrine/Migrations/InlineParameterFormatter.php

```diff
@@ -91,7 +91,5 @@ private function parameterToString($value) : string
         if (is_bool($value)) {
             return $value === true ? 'true' : 'false';
         }
-
```

Comment:
any particular reason for removing this?

Reply:
Yes, it's unreachable according to the docblock. If I leave it, phpstan complains with `Unreachable statement - code above always terminates.`

Reply:
:+1: